.github/meta.yaml

```diff
@@ -72,7 +72,7 @@ outputs:
         - python-kaleido >=0.1.0
         - matplotlib-base >=3.3.3
         - seaborn >=0.11.1
-        - ipywidgets >=7.5, <8.0.5
+        - ipywidgets >=7.5
```

Comment:
Looks like ipywidgets must have fixed whatever was causing the errors we saw back in March, if we're able to unpin and have no tests fail. However, I think if a user happened to have ipywidgets v8.0.5 installed in the environment where they're installing evalml, the issues would still exist. 

Could you install ipywidgets 8.0.5 and the following versions locally, to see if they still error? If so, we should exclude those specific versions.

Reply:
The tests passed for v.8.0.5-7 as well!

Reply:
👀 surprising, but glad it'll work 🚢